src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -121,35 +122,37 @@ const FIELD_MAPPING_HELP_TEXT = {
 
 export default class geTemplateBuilderFormFields extends LightningElement {
 
-    // Expose labels to template
-    CUSTOM_LABELS = GeLabelService.CUSTOM_LABELS;
 
-    isInitialized;
     @api previousSaveAttempted;
     @api selectedFieldMappingSet;
-    @track isLoading = true;
-
     @api formSections;
     @api activeFormSectionId;
     @api sectionIdsByFieldMappingDeveloperNames;
+
     @track objectMappings;
-    @track isAllSectionsExpanded = false;
-    objectMappingNames = [];
+    @track pageLevelError = {};
 
-    @track isReadMoreActive = false;
-    @track hasErrors = false;
-    @track errors;
+    isInitialized;
+    isLoading = true;
+    isAllSectionsExpanded = false;
+    isReadMoreActive = false;
 
+    // Expose labels to template
+    CUSTOM_LABELS = GeLabelService.CUSTOM_LABELS;
+    objectMappingNames = [];
     // identifiers for use with querySelectors
     LOCATORS = {
         FIELD_BUNDLES_SECTION_ID,
         FORM_FIELDS_SECTION_ID,
         ADVANCED_FIELDS_SECTION_ID
     };
+    sourceObjectFieldInfo;
```

Comment:
Could we rename to `sourceObjectFieldsInfo` or `sourceObjectFieldsDescribe` or `sourceObjectFields`?

Reply:
I like `sourceObjectFieldsDescribe` - it's descriptive!

Reply:
done!